tpgtools/README.md

```diff
@@ -6,7 +6,7 @@ to describe the available types, and `tpgtools` uses those to construct
 Terraform resource schemas.
 
 ## Usage
```

Comment:
We should probably add some instructions on how to use the samples folder here, if you want to do that in a follow up or something

Reply:
Going to do that in a follow up since things are changing soon